packages/subapp-web/lib/init.js

```diff
@@ -68,6 +70,13 @@ ${inlineRuntimeJS}
       runtimeEntryPoints.forEach(ep => {
         context.user.includedBundles[ep] = true;
       });
+
+      if (metricReport.enable && metricReport.reporter) {
+        context.user.emitter = util.getEventEmiiter(metricReport.reporter);
+      } else {
+        // eslint-disable-next-line
+        console.warn(`Warning: Metric reporting for ssr not enabled or no reporter specified.`);
```

Comment:
isn't this going to log for every request?

Reply:
oh yes, you are right. I should probably move this to **subapp-server**. Was trying to avoid changes to another package.